src/main/kotlin/no/nav/familie/ef/sak/Vedtaksbrev/BrevClient.kt

```diff
@@ -24,9 +26,22 @@ class BrevClient(@Value("\${FAMILIE_BREV_API_URL}")
     }
 
     fun genererBrev(målform: String, malnavn: String, request: BrevRequest): ByteArray {
+        val url = URI.create("$familieBrevUri/api/${ef}/avansert-dokument/$målform/$malnavn/pdf")
+
+        val streng = objectMapper.readTree(request.lagBody())
+
+        return postForEntity(url, streng, HttpHeaders().medContentTypeJsonUTF8())
+    }
+
+    fun genererBrev(målform: String? = "bokmaal" , malnavn: String? = "innvilgetVedtakMVP", request: JsonNode): ByteArray {
```

Comment:
målform og malnavn - er dette egentlig enumer? Eller er det "dynamisk-on-the-fly"-generert ?

Reply:
Vi fikser dette i neste pr ( kaller det "opprydding")